src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -881,15 +881,17 @@ public static uint Crc32C(uint crc, ulong data)
                 return Crc32.Arm64.ComputeCrc32C(crc, data);
             }
 
-            return Crc32Fallback.Crc32C(crc, data);
+            return Crc32C(Crc32C(crc, (uint)(data)), (uint)(data >> 32));
         }
 
         private static class Crc32Fallback
         {
-            // Pre-computed CRC-32 transition table.
+            // CRC-32 transition table.
             // While this implementation is based on the Castagnoli CRC-32 polynomial (CRC-32C),
             // x32 + x28 + x27 + x26 + x25 + x23 + x22 + x20 + x19 + x18 + x14 + x13 + x11 + x10 + x9 + x8 + x6 + x0,
-            // this version uses reflected bit ordering, so 0x1EDC6F41 becomes 0x82F63B78u
+            // this version uses reflected bit ordering, so 0x1EDC6F41 becomes 0x82F63B78u.
+            // This is computed lazily so as to avoid increasing the assembly size for data that's
+            // only needed on a fallback path.
             private static readonly uint[] s_crcTable = Crc32ReflectedTable.Generate(0x82F63B78u);
```

Comment:
If this is the last usage of `Crc32ReflectedTable`, could we copy over the method (or transform to generated table as done in `Crc32.Table`) and delete `Crc32ReflectedTable.cs`?

Reply:
> or transform to generated table as done in Crc32.Table

I explicitly opted not to do that, as noted in the comment.

> could we copy over the method and delete Crc32ReflectedTable.cs

We _could_.  I don't have a strong opinion on doing so.